### PR TITLE
fix: MultiSelect/TreeSelect widget deselects on clicking again 

### LINKED ISSUE #16948: Select widget deselects on clicking again
The select widget works weird on prod as well, selecting the widget and then clicking on it sometimes deselects the widget in previous and this version of react, this is because of the combination of our keyboard navigation on the property pane that we kick off when a widget is selected vs the select widgets own keyboard navigation support. This is a hunch based on how the widget behaves but needs more investigation coz its a third party widget.

_Originally posted by @marks0351 in https://github.com/appsmithorg/appsmith/issues/16257#issuecomment-1236882805_

### PR DESCRIPTION
**Pull request by jsartisan**
## Description
This PR fixes deselecting of select widget on clicking it again.

Fixes #16948

## Type of change
- Bug fix 
- 
## How Has This Been Tested?
- Manually test

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/widgets/useDropdown.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const BackDropContainer = styled.div`
   background: transparent;
   top: 0;
   left: 0;
+  display: none;
```
Comment (Tooluloope) [inline app/client/src/widgets/useDropdown.tsx:14]: @jsartisan, `useDropdown` only applies to MultiSelect and TreeSelect.

it's not been used in Select Widget.

Comment (jsartisan) [reply app/client/src/widgets/useDropdown.tsx:14]: Yea the title of the issue is wrong. the issue is in multiselect/treeselects

Comment (Tooluloope) [reply app/client/src/widgets/useDropdown.tsx:14]: Can you change it as well as the [issue](https://github.com/appsmithorg/appsmith/issues/16948) so it's more descriptive?

Comment (jsartisan) [reply app/client/src/widgets/useDropdown.tsx:14]: Done.

**Review by Tooluloope — APPROVED** [review]

**Pull request merged**